content/using/running/vere.md

```diff
@@ -0,0 +1,633 @@
+```
+
+### `-c, --pier PIER`
+
+When booting a ship, name its pier as `PIER` rather than the usual `@p`.
+
+- Undocked: `urbit -w sampel-palnet -k /path/to/my.key -c foobar`
+
+The pier folder will be named `foobar` as a result of the above command.
+
+### `-C, --memo-cache-limit LIMIT`
+
+This sets the maximum number of entries for the memoization cache (used by the
+`~+` rune and a few other things). Zero means uncapped, otherwise the specified
+number of entries.
+
+- Undocked: `urbit -C 50000 [pier]`
+- Docked: `[pier]/.run -C 50000`
+
+### `-d, --daemon`
+
+Run the ship in daemon mode. The process will be detached from the terminal
+(this implies `-t`).
+
+- Undocked: `urbit -d [pier]`
+- Docked: `[pier]/.run -d`
+
+### `-D, --replay`
+
+Recompute a ship's state from events rather than loading its checkpoint. This
+option is currently a no-op, and doesn't do anything. It may be revived in the
+future or it may be removed.
+
+### `-e, --ethereum URL`
+
+Use the Ethereum gateway at `URL` rather than the default.
+
+- Undocked: `urbit -e https://the.url [pier]`
+- Docked: `[pier]/.run -e https://the.url`
+
+### `-F, --fake SHIP`
+
+Boot a new fake development ship. Real networking is disabled, and instead the
+fake ship can only communicate with other fake ships on the local machine.
+
+```
+urbit -F zod
+```
+
+### `-g, --gc`
+
+Set GC flag, dumping garbage collector logs to stdout.
+
+- Undocked: `urbit -g [pier]`
+- Docked: `[pier]/.run -g`
+
+### `-G --key-string STRING`
+
+Boot a new ship with private key `STRING`.The private key is `@uw` format. See
+`-k` for an alternative.
+
+- Undocked: `urbit -w sampel-palnet -G 0w50w.642KA.glYh8.RvaRp.6TA35.3XRYN.ULCI6.UmcsZ.5rGvG.3dvFt.E6rMF.mZnd~.3O6QB.7LZ1f.-ukYN.z2fcY.Ecviv.LiZEg.8w0sp`
+
+### `--http-port PORT`
+
+Set the HTTP port to `PORT`. This is the port you'd insecurely access Landscape
+on. Note this is for insecure HTTP only, see `--https-port` for HTTPS.
+
+- Undocked: `urbit --https-port 8085 [pier]`
+- Docked: `[pier]/.run --https-port 8085`
+
+### `--https-port PORT`
+
+Set the HTTPS port to `PORT`. This is the port you'd securely access Landscape
+on. Note that this requires an SSL certificate to be configured, see the
+[Configuring SSL guide](https://urbit.org/using/os/basics#configuring-ssl) for
+details. Note that if your ship is behind a reverse proxy (nginx, Caddy, etc)
+and an SSL certificate is configured _there_, you'd not usually enable HTTPS on
+the ship itself.
+
+Note this is for secure HTTPS only, see `--http-port` for insecure HTTP.
+
+- Undocked: `urbit --https-port 8443 [pier]`
+- Docked: `[pier]/.run --https-port 8443`
+
+### `-I, --inject FILE`
+
+Inject the event stored in `FILE` jamfile.
+
+- Undocked: `urbit -I /path/to/file.jam [pier]`
+- Docked: `[pier]/.run -I /path/to/file.jam`
+
+### `-j, --json-trace`
+
+Create JSON trace files in `[pier]/.urb/put/trace`.
+
+- Undocked: `urbit -j [pier]`
+- Docked: `[pier]/.run urbit -j`
```

Comment:
This runs a tracing profiler, and produces JSON output compatible with `chrome://tracing`.